README.md

```diff
@@ -52,7 +52,7 @@ Maven:
 <dependency>
   <groupId>org.nuxeo.client</groupId>
   <artifactId>nuxeo-java-client</artifactId>
-  <version>1.0</version> <!-- (or <version>2.1</version>) -->
+  <version>1.0</version> <!-- (or <version>2.3-SNAPSHOT</version>) -->
```

Comment:
You can replace <version>1.0</version> by <version>2.3-SNAPSHOT</version> and remove the comment